developing/languages/README.md

```diff
@@ -17,3 +17,39 @@ While Truss may have opinions on particular languages and strategic reasons for
 - Python
 - Java
 - C/C++
+
+## Language Resources
+
+### Go
+If you are new to Go, you should work your way through all of these resources (in this order, ideally):
```

Comment:
Hmm - I am not sure everyone needs to read everything. I know I haven't read all of those. Many folks learn best by reading a brief intro and then trying to do things with a language, rather than a large chunk of readying up front.

Reply:
I did crib this from Milmove resources as well, I can change because your suggestion does make more sense.